src/utils/worklets/communities.cljs

```
@@ -0,0 +1,33 @@
+(ns utils.worklets.communities
+  (:require [goog.object :as gobj]
+            [react-native.utils :as utils]
+            [taoensso.timbre :as log]))
+
+(def worklets (js/require "../src/js/worklets/communities.js"))
+
+(defn- transform-args
+  [f]
+  (fn [& args]
+    (apply f (map utils/kebab-case-map->camelCase-obj args))))
+
+(defn- worklet-wrapper
+  [worklet-name]
+  (if-let [worklet-fn (gobj/get worklets worklet-name)]
+    (transform-args worklet-fn)
+    (log/error "Non-existing worklet!"
```

Here I would throw an error right away instead of logging, similar to how we changed Malli schemas to throw during app setup in case they are invalid. We know that if `worklet-wrapper` received a non-existent worklet name it's a developer mistake and it's risky to proceed and merge by mistake. The def'ed var will be nil because `log/error` returns nil. In release builds the caller of these vars won't check if they are nullable and thus we would risk exceptions in production.